bc_obps/service/data_access_service/user_operator_service.py

```diff
@@ -0,0 +1,52 @@
+from uuid import UUID
+from service.data_access_service.user_service import UserDataAccessService
+from service.data_access_service.operator_service import OperatorDataAccessService
+from registration.models import Operator, User, UserOperator
+from django.db import transaction
+
+
+from registration.models import (
+    Operator,
+    UserOperator,
+)
+
+
+class UserOperatorDataAccessService:
+    def get_user_operator_by_id(user_operator_id: UUID):
+        # brianna this is the optimized one?
+        return UserOperator.objects.select_related('operator').get(id=user_operator_id)
+        # return UserOperator.objects.get(id=user_operator_id)
+
+    def get_admin_users(operator_id: UUID, desired_status: UserOperator.Statuses):
+        operator = Operator.objects.get(id=operator_id)
+        user_operators = UserOperator.objects.filter(
+            operator=operator, role=UserOperator.Roles.ADMIN, status=desired_status
+        )
+        admin_users = User.objects.filter(user_guid__in=user_operators.values('user_id'))
+
+        return admin_users
+
+    def get_an_operators_user_operators_by_user_guid(user_guid: UUID):
+        user = UserDataAccessService.get_user_by_guid(user_guid)
+        operator = (
+            UserOperator.objects.select_related("operator")
+            .exclude(status=UserOperator.Statuses.DECLINED)
+            .get(user=user.user_guid)
+            .operator
+        )
+        user_operator_list = UserOperator.objects.select_related("user").filter(
+            operator_id=operator, user__business_guid=user.business_guid
```

Comment:
`operator_id=operator` will always be false since they're of different types

Reply:
:scream:

Reply:
This is interesting because this wasn't a refactor whoopsie; it's always been like this.